calico-cloud_versioned_docs/version-3.17/image-assurance/understanding-scan-results.mdx

```diff
@@ -2,111 +2,96 @@
 description: Understand scan results in Manager UI.
 ---
 
-# Understand scan results in Manager UI
+# Understand image scan results in Manager UI
 
-In the left navbar in Manager UI, click **Image Assurance**, **Scan Results**.
+This article provides help using Image Assurance in Manager UI:
 
-**Scan Results tab**
+- [All Scanned Images dashboard](#all-scanned-images-dashboard)
+  - [Scan settings panel](#scan-settings-panel)
+  - [Exclude vulnerabilities from scan results](#exclude-vulnerabilities-from-scan-results)
+  - [Export scan results](#export-scan-results)
+  - [Troubleshooting](#troubleshooting)
+- [Running images tab](#running-images-tab)
 
-The left panel lists scanned images (assuming you set up the CLI scanner to report scan results to {{prodname}}).
+- [Vulnerability exceptions dashboard](#vulnerability-exceptions-dashboard)
 
-The right pane lists:
+## All Scanned Images dashboard
 
-- Number of images grouped by Fail, Warn, Pass, and Unknown (the CLI scanner has not yet reported results)
-- High Risk Repos (top five repos based on total Fails)
-- Image details (when you have an image selected from the left panel list) lists associated CVEs, if any
-    - For each CVE listed, you can view the details of the vulnerability or add an exception to ignore the vulnerability (see [Create vulnerability exceptions](#create-vulnerability-exceptions)). 
+In the left navbar in Manager UI, click **Image Assurance**, **All Scanned Images**.
 
-**Runtime tab**
+This dashboard lets you:
+- View results of image scanning and assess vulnerabilities prior to runtime
+- View running images in your cluster and remediate CVEs
 
-The left panel lists images running in your cluster. Because you can see the CVEs associated with running pods, you can assess pod vulnerability. The columns, **Clusters** and **Running Instances**, show the number of running instances in clusters that are connected to {{prodname}}.
+### Scan results tab
 
-The right panel lists:
+In this panel, you can:
 
-- Number of running pods that have images with scan results of Fail, Warn, Pass, and Unknown (the CLI scanner has not yet reported results)
-- High Risk Namespaces (top five namespaces based on total Fails)
-- Runtime details (when you have an image selected from the left panel list) shows the clusters and namespaces the selected image is running in
+- View all scanned images for CVEs
+- Change default scan settings
+- Exclude vulnerabilities from scan results
```

Comment:
The new doc doesn't seem to have any content related to package-centric scan results. I think at least it should include the following content:

- When clicking on a vulnerable image, how are data on the right panel organized.
- What would happen if we create/delete an exception applied to one of the  vulnerabilities of one of its dependencies.

Reply:
@xiumozhan The UI docs are here: Understanding scan results.